tests/test_bgp_neighbor_af.rb

```diff
@@ -117,6 +184,16 @@ def test_nbrs_with_masks
     @@matrix.each do |k, v|
       asn, vrf, nbr, af = v
       nbr += (nbr[/:/]) ? '/64' : '/16'
+      if platform == :ios_xr
+        # XR does not do slash masks and neighbors
+        # so make sure to catch the error
+        assert_raises(ArgumentError, 'Neighbors do not support /') do
+          RouterBgpNeighborAF.new(asn, vrf, nbr, af, true)
+        end
+        # Finish the test case with regular neighbor
+        # to avoid skipping whole testcase
```

Comment:
This seems dubious - this test case is explicitly `test_nbrs_with_masks` - doesn't look like it does anything that isn't already covered by `test_nbr_af_create_destroy` above and I think proceeding with a non-mask neighbor is not in the spirit of the test. Would be better to 'return' after the above assertion.